src/components/BorrowerProfile/LoanUse.vue

```diff
@@ -3,8 +3,9 @@
 		{{ loanUseFiltered }}
 		<kv-text-link
 			v-if="loanId && showLearnMore"
-			:to="`/lend/${loanId}`"
+			:to="showLoanDetails ? '' : `/lend/${loanId}`"
 			v-kv-track-event="['Lending', 'click-Read more', 'Learn more', loanId]"
+			@click="showLoanDetails({loanId})"
```

Comment:
Will clicking this link error if `showLoanDetails` is not defined?

Reply:
No, there's a default function that sets itself when it's not defined.  That default function is empty and does nothing.

Reply:
I'm actually changing it now also -- so that it's not a function anymore but a boolean that causes the component to emit an event and pass the loan id up to the parent.  It's cleaner than this mess.